.github/CODEOWNERS

```diff
@@ -2,13 +2,14 @@
 # below ones takes precedence over the upper ones
 
 # Global owners
-*                       @nazar-pc @rg3l3dr
-
-# Scripts
-/scripts/               @ImmaZoni @nazar-pc @rg3l3dr
-
-# Protocol
-/crates/                @liuchengxu @nazar-pc @rg3l3dr
-
-# Farmer
-/crates/subspace-farmer/ @ozgunozerk @liuchengxu @nazar-pc @rg3l3dr
+*                           @nazar-pc @rg3l3dr
+
+/crates                     @liuchengxu @nazar-pc @rg3l3dr
+/crates/pallet-*            @liuchengxu @nazar-pc @rg3l3dr @vedhavyas
+/crates/sp-*                @liuchengxu @nazar-pc @rg3l3dr @vedhavyas
+/crates/subspace-archiving  @liuchengxu @i1i1 @nazar-pc @rg3l3dr
+/crates/subspace-farmer     @i1i1 @nazar-pc @rg3l3dr
+/crates/subspace-networking @nazar-pc @rg3l3dr @shamil-gadelshin
```

Comment:
"Unknown owner on line 12: make sure @shamil-gadelshin exists and has write access to the repository
…networking @nazar-pc @rg3l3dr @shamil-gadelshin"

Just want to ensure this isn't a typo and is a false positive error

Reply:
I expect Shamil to accept org invitation before merging :slightly_smiling_face: